src/main/java/zeenea/connector/property/PropertyDefinition.java

```diff
@@ -129,18 +143,19 @@ public static InstantPropertyDefinition instant(String code) {
    * @param label the label of the property definition
    * @return a new InstantPropertyDefinition instance
    */
+  @Deprecated(since = "2.3.0", forRemoval = true)
```

Comment:
Itou : 
```suggestion
  Et là également : 
```suggestion
  
  @Deprecated(
      since =
          "Deprecated since version 2.3.0, use instant(name) instead. Scheduled for removal in version 3.0.0.",
      forRemoval = true)
```
```